opensearch/Cargo.toml

```diff
@@ -40,22 +40,27 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 serde_with = "3"
 void = "1.0.2"
-aws-credential-types = { version = ">= 0.53", optional = true }
-aws-sigv4 = { version = ">= 0.53", optional = true }
-aws-types = { version = ">= 0.53", optional = true }
+aws-credential-types = { version = ">= 0.57", optional = true }
+aws-sigv4 = { version = ">= 0.57", optional = true }
+aws-smithy-runtime-api = { version = ">= 0.57", optional = true, features = ["client"]}
```

Comment:
This dependency is very suspicious [1]:

> aws-smithy-runtime-api
> APIs needed to configure and customize the Smithy generated code.

We don't use any Smithy generated code here, do we?


[1] https://crates.io/crates/aws-smithy-runtime-api

Reply:
It's where the `Identity` type used for passing the credentials into `aws-sigv4` is defined, as is not re-exported anywhere else. It's due to `aws-sigv4` etc being part of the `AWS SDK for Rust` which is generated from Smithy via `smithy-rs`

Reply:
Got it, thanks @Xtansia , LGTM to me but may be @dblock could have valuable feedback as well

Reply:
🤷